applications/ParticleMechanicsApplication/python_scripts/apply_mpm_slip_boundary_process.py

```diff
@@ -26,12 +26,6 @@ def __init__(self, Model, settings ):
         # Note that the model part employed here is supposed to only have slip "conditions"
         KratosMultiphysics.NormalCalculationUtils().CalculateOnSimplex(self.model_part, self.model_part.ProcessInfo[KratosMultiphysics.DOMAIN_SIZE])
 
-        # Mark the nodes and conditions with the appropriate slip flag
-        #TODO: Remove the IS_STRUCTURE variable set as soon as the flag SLIP migration is done
-        for condition in self.model_part.Conditions: #TODO: this may well not be needed!
```

Comment:
@VeronikaSinger Could you please check with the SLIP BCs? I think we still need it right?

Reply:
it is still needed for the check in mpm_boundary_rotation_utility.h whether is_penalty is true or false